policy/release/labels.rego

```diff
@@ -0,0 +1,155 @@
+#
+# METADATA
+# description: >-
+#   Check if the image has the expected labels set. The rules in this package
+#   distinguish file-based catalog (FBC) images from all other images. When
+#   checking an FBC image, a policy rule may use a different set of rule data.
+#
+package policy.release.labels
```

Comment:
I will assume that I don't need to look at how you defined the label-based interactions.

Reply:
As a reviewer of this PR? Or do you mean in a broader sense as a user/author reading the docs?

Reply:
As a reviewer.